reporter/reporter.go

```diff
@@ -0,0 +1,65 @@
+package reporter
+
+import (
+	"context"
+	"fmt"
+	pb "github.com/beaujr/nmap_prometheus/proto"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"log"
+	"time"
+)
+
+type Reporter struct {
+	address string
+}
+
+func NewReporter(address string) Reporter {
+	return Reporter{address: address}
+}
+
+func (r *Reporter) Address(items []pb.AddressRequest) error {
```

Comment:
warning: exported method Reporter.Address should have comment or be unexported (golint)